### ISSUE TITLE
Cannot update WordPress (6.5.2 > 6.5.3)

### ISSUE THREAD

**Issue by dbushell** [Automattic/studio#100]
### Quick summary

When I click _"Update to version 6.5.3"_ in **Dashboard > Updates** I get taken to a page that says:

> Another update is currently in progress.

Nothing else happens and I am unable to update. The _"WordPress 6.5.3 is available! Please update now."_ notice and _"Updates (1)"_ badge remains.

I've tried to stop/start both the site and the Studio app but I always get the same message attempting to update.

If I create a new project it's already on 6.5.3 so I can't test that. The site I'm trying to update is almost new with very little content or settings changed.

### Steps to reproduce

1. Open project on 6.5.2
2. Click "Update to version 6.5.3" from the Dashboard

### What you expected to happen

The update completes

### What actually happened

I get stuck with a message "Another update is currently in progress."

### Impact

One

### Available workarounds?

No but the app is still usable

### Platform

Mac Silicon

**Comment by dbushell** [comment]
Maybe related, trying to view the `wp_options` table with [SQLite VS Code extension](https://marketplace.visualstudio.com/items?itemName=alexcvzz.vscode-sqlite) errors:

```
Runtime error near line 4: database disk image is malformed (11)
```

Using [DB Browser](https://sqlitebrowser.org/) gets stuck _"loading..."_:

![sqlite](https://github.com/Automattic/studio/assets/998922/2a0f26e6-6431-4e5c-aa23-02bdcf118330)

Is the database corrupted in some way?

I'm not getting any errors or other issues in the WP Admin. Everything else seems to be working.

**Comment by dbushell** [comment]
Sorry I spoke too soon. I'm getting issues trying to use WP Admin adding a menu. I enabled `WP_DEBUG_LOG` and it looks like I'll have to start from scratch.

Attached full log: [debug.log](https://github.com/Automattic/studio/files/15245821/debug.log)

**Comment by dbushell** [comment]
I started a new project and copied the theme over. I tried to create a menu (`wp-admin/nav-menus.php`) but the `POST` request timed out. No errors logged though. I don't know if these issues are related.

Anway, uninstalled WordPress Studio and moved back to Docker. Now it's working as expected.

**Comment by danielbachhuber** [comment]
Thanks for reporting, @dbushell ! We'll get this fixed up 😊

**Comment by wojtekn** [comment]
It is potentially the same issue as https://github.com/Automattic/dotcom-forge/issues/6891

**Comment by danielbachhuber** [comment]
Also related https://github.com/WordPress/playground-tools/issues/178

**Comment by liviopv** [comment]
📌 REPRODUCTION RESULTS
- Tested on Studio - Replicated

📌 FINDINGS/SCREENSHOTS/VIDEO
While triaging https://github.com/Automattic/studio/issues/71#top, I ran into this issue in my site and the update got stuck.
I ended up deleting the site and spinning a new one.

📌 ACTIONS
- Marking this as triaged considering it's on your radar @wojtekn @danielbachhuber but please ping if you need more help with testing.

**Comment by SiobhyB** [comment]
@dbushell, we merged a fix today and I believe that fix will be available in v1.0.3 (scheduled for release on June 3rd). I'm just seeking confirmation on that date for you, though. If you give the app another go and encounter any other issues or have general feedback, we'd love to hear it. Thank you again for reporting and helping us make the app better. 🙇‍♀️

**Comment by wojtekn** [comment]
@dbushell in case you would you like to test it using the latest beta version, I'm sharing links to fresh builds: [Mac Intel](https://href.li/?https://cdn.a8c-ci.services/studio/studio-darwin-x64-v1.0.3-beta2.app.zip), [Mac Apple Silicon](https://href.li/?https://cdn.a8c-ci.services/studio/studio-darwin-arm64-v1.0.3-beta2.app.zip) and [Windows](https://href.li/?https://cdn.a8c-ci.services/studio/studio-win32-v1.0.3-beta2.exe). Note that this version has not been fully tested yet.
